packages/sanes-chrome-extension/src/utils/test/LinkMock.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { history } from '../../store/reducers';
+
+interface Props {
+  readonly to: string;
+  readonly children: React.ReactNode;
+}
+
+const LinkMock = ({ to, children }: Props): JSX.Element => {
+  const onClick = (): boolean => {
+    history.push(to);
+    return false;
+  };
+
+  return (
+    <a href={to} onClick={onClick}>
```

Comment:
This is odd. Why in wallet-demo we did not need anything like this?

Reply:
I do not know. But `Simulate.click` doesn't redirect to another route. Odd I know, but I can not understand why. Components looks same.